### PR TITLE
Removed Serialization tests for SQLClient types

### LINKED ISSUE #38949: Remove Serialization tests for SQLClient types now that they no longer live in the dotnet/runtime repo
referencing @stephentoub's comment here https://github.com/dotnet/runtime/pull/38948#discussion_r451742184

Now that SqlClient is no longer part of dotnet/runtime repo we should remove the serialization tests for those types. The comment on the PR above pretty much sumps up which section needs to be removed from the serialization test data.

### PR DESCRIPTION
**Pull request by yujinnee**
Fixes [#38949](https://github.com/dotnet/runtime/issues/38949)

Removed OperationAbortedException because it was moved out of System.Data into SqlClient and so the testing belongs in the M.D.SqlClient repo. Other serialization tests for SQLClient types remain in this repo ,so I didn't removed them.

### CODE REVIEW

File: src/libraries/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (jkotas) [inline src/libraries/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs:1]: Could you please undo this whitespace change?

**Comment by jkotas** [comment]
You should be also able to delete reference to System.Data.SqlClient from the project file.

**Comment by focusonmx** [comment]
@jkotas  I deleted white space and reference to System.Data.SqlClient!

**Comment by danmosemsft** [comment]
Why remove this and not tests in the same file for eg SqlException?

**Comment by focusonmx** [comment]
@danmosemsft  I referred to [this comment](https://github.com/dotnet/runtime/issues/38949#issuecomment-656075897). Can it be a proper answer to your question??

**Comment by jkotas** [comment]
I meant to delete it from here: https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime.Serialization.Formatters/tests/System.Runtime.Serialization.Formatters.Tests.csproj#L40

**Comment by jkotas** [comment]
Also, the build errors suggest that there are more SqlClient-related types that should be deleted from this test.

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=42016) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=42016) before we can accept your contribution.<br/><br/>:x: focusonmx [sign now](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=42016)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/runtime?pullRequest=42016) it.</sub>

**Comment by jkotas** [comment]
Could you please also delete SqlClient references from https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime.Serialization.Formatters/tests/System.Runtime.Serialization.Formatters.Tests.csproj#L40 ?

**Comment by focusonmx** [comment]
To delete that reference, should I also remove the serialization tests related to SQLClinet type in other cs file in System.Runtime.Serialization.Formatters/tests ?

**Comment by Wraith2** [comment]
Yes, anything that needs SqlClient can be removed. Anything which only needs System.Data.Common can be retained.

**Review by jkotas — APPROVED** [review]

**Pull request merged**